### PR TITLE
Mention VSCode Plot Debugging in docs

### LINKED ISSUE #5057: vscode: mention plots debugging tools
From https://github.com/iterative/vscode-dvc/issues/5087:

> 3\. Can we improve documentation to mention existing features for error messages?

### PR DESCRIPTION
**Pull request by julieg18**
Adds video on plots debugging to `/doc/user-guide/experiment-management/visualizing-plots?tab=VSCode-Extension#visualizing-plots`

## Demo

~https://github.com/iterative/dvc.org/assets/43496356/7180c824-c003-46ef-bfb0-1cbf41b20c78~

Screenshot 2024-02-02 at 2 51 00 PM

Fixes #5057

### CODE REVIEW

File: content/docs/user-guide/experiment-management/visualizing-plots.md [hunk 1]
```diff
@@ -268,6 +268,10 @@ view.
 
 ![VS Code Comparison](/img/dvclive-vscode-compare.png)
 
+The extension can also make plots debugging easier.
+
+https://youtu.be/XH0v3x7lBDg?si=6YheW-ocPU9MMoIq
```
Comment (julieg18) [inline content/docs/user-guide/experiment-management/visualizing-plots.md:273]: We don't really mention plots debugging anywhere that I could find in the docs and I eventually chose this spot, but maybe there are other places?

Comment (dberenbaum) [reply content/docs/user-guide/experiment-management/visualizing-plots.md:273]: This seems pretty hidden to me. Could we either put it under the other VS Code video in this page or create a new section for debugging plots?

Comment (julieg18) [reply content/docs/user-guide/experiment-management/visualizing-plots.md]: Good point! I've updated the video to be under the defining plots video: 

Screenshot 2024-02-02 at 2 51 00 PM

**Review by dberenbaum — APPROVED** [review]
Thanks @julieg18!

**Pull request merged**